src/js/videojs.wavesurfer.js

```diff
@@ -14,10 +14,11 @@ import videojs from 'video.js';
 import WaveSurfer from 'wavesurfer.js';
 
 const Plugin = videojs.getPlugin('plugin');
+const Tech = videojs.getComponent('Tech');
```

Comment:
@mfairchild365 I cannot find any references to this const so it's safe to remove this Tech import right?

Reply:
Yes. I actually meant to remove that. The new tech was originally extending `Tech`, but I changed it to extend the 'HTML5' tech. Good catch.

Reply:
cool. cleaned it up a little and moved it to a separate module. Will release as 2.1.0, probably today. Cheers.